OMDbSharp/OMDbSharp.nuspec

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Why was the nuspec deleted? It's needed for deployment?

Reply:
My bad, I've missed the file when rebuilding the project. I'll have it added back in the next commit.